blynk_ncp/src/zephyr_ota.c

```diff
@@ -104,13 +100,13 @@ bool rpc_client_otaUpdateWrite_impl(uint32_t offset, rpc_buffer_t chunk, uint32_
     bool crcOK = (calcCRC32(chunk.data, chunk.length, 0) == crc32);
 
     #if defined(BLYNK_DEBUG_ALL)
```

Comment:
Are we need this check if we use standard zephyr logging and config log level in project?

Reply:
Ye, probably need to remove. This is not configurable anywhere